README.md

```diff
@@ -104,6 +104,11 @@ should always be set to `false` (the default) in production environments.
 The `project` parameter is the name of the AWX project that contains the job
 templates that will be used to run the playbooks.
 
+The autoheal service performs a periodic job status check against AWX server,
```

Comment:
In the rest of the documentation we write _auto-heal_, with a dash, can you do the same here?